feature/gribi/ate_tests/hierarchical_weight_resolution_test/hierarchical_weight_resolution_test.go

```diff
@@ -259,51 +280,55 @@ func (a *attributes) configInterfaceDUT(t *testing.T, d *ondatra.DUTDevice, p *o
 	t.Helper()
 
 	i := &oc.Interface{Name: ygot.String(p.Name())}
-
-	if a.numSubIntf > 0 {
-		i.Description = ygot.String(a.Desc)
-		i.Type = oc.IETFInterfaces_InterfaceType_ethernetCsmacd
-		if *deviations.InterfaceEnabled {
-			i.Enabled = ygot.Bool(true)
-		}
-	} else {
-		i = a.NewOCInterface(p.Name())
+	i.Description = ygot.String(a.Desc)
```

Comment:
Is Line 283- Line 286 valid for a.numSubIntf==0?

Reply:
I don't see why they wouldn't be valid (it's just a basic Ethernet interface config -> description/type/speed/enabled), I would assume it would (and should) be applicable for all Ethernet interfaces regardless of number of subinterfaces.

But if there are any knows issues or limitations - please le me know

Reply:
Updated per our conversation.